src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projecthome/viewmodel/ProjectHomeViewModel.kt

```diff
@@ -0,0 +1,15 @@
+package org.wycliffeassociates.otter.jvm.app.ui.projecthome.ViewModel
+
+import org.wycliffeassociates.otter.common.data.dao.Dao
+import org.wycliffeassociates.otter.common.data.model.Project
+import org.wycliffeassociates.otter.common.domain.GetProjectsUseCase
+import tornadofx.*
+import org.wycliffeassociates.otter.common.data.model.Language
+import io.reactivex.Observable
+import java.util.*
+
+class ProjectHomeViewModel: ViewModel() {
+    // TODO(need repo)?
+    var projectUseCase: GetProjectsUseCase? = null
+    val projects : Observable<List<Project>>? = projectUseCase?.getProjects()
```

Comment:
How can we avoid a lateinit on projectusecase and prevent it from being nullable?

Can the viewmodel work if projectusecase is null? Nope, which then means the ability from any further things that now have to inherit nullable types are guaranteed to not work either. We need to avoid this even being possible.